CHANGELOG.md

```diff
@@ -8,7 +8,11 @@ and this project adheres to [Semantic Versioning][].
 [keep a changelog]: https://keepachangelog.com/en/1.0.0/
 [semantic versioning]: https://semver.org/spec/v2.0.0.html
 
-## [0.0.15] - tbd
+## [0.1.0] - tbd
```

Comment:
I wouldn't release 0.1.0 just yet, there are too many changes around storage, data models and transformations that needs to happen, wdyt? @melonora @LucaMarconato @kevinyamauchi

Reply:
Maybe for multiple tables let's make 0.1.0, but we could also wait for a all the big PRs being merged. I changed for the moment.